packages/mcp-registry-client/src/mcp-registry-client.ts

```diff
@@ -75,32 +75,42 @@ export class MCPRegistryClient {
         });
         const response = await this.#fetch(resolvedURL);
         if(!response.ok) throw new Error(`Failed to fetch servers for registry ${this.#baseURL}: ${response.statusText} - (URL: ${resolvedURL})`);
-        const body = await response.json()
-        return body as components['schemas']['ServerList'];
+        return (await response.json()) as paths['/v0/servers']['get']['responses'][200]['content']['application/json'];
```

Comment:
This cast already existed but it is a bit dangerous

Reply:
I can't find a way to do it nicely :/

Reply:
In the repository https://github.com/kortex-hub/mcp-registry-types we could also generate zod types from the OpenAPI and use them to safe parse the results.